lib/DataIntegrityProof.js

```diff
@@ -292,7 +308,11 @@ export class DataIntegrityProof extends LinkedDataProof {
       if(!verified) {
         throw new Error('Invalid signature.');
       }
-
+      if(proof.created) {
```

Comment:
Need to handle empty string too, I believe:

```suggestion
      if(proof.created !== undefined) {
```